api.md

```diff
@@ -850,10 +851,11 @@ partition across
 
 #### Optional Arguments [](detach_data_node-optional-arguments)
 
-| Name         | Description                            |
-|--------------|----------------------------------------|
-| `hypertable` | Name of the distributed hypertable where the data node should be detached. If NULL, the data node will be detached from all hypertables. |
-| `force`      | Force detach of the data node even if that means that the replication factor is reduced below what was set. Note that it will never be allowed to reduce the replication factor below 1 since that would cause data loss.         |
+| Name          | Description                            |
+|---------------|----------------------------------------|
+| `hypertable`  | Name of the distributed hypertable where the data node should be detached. If NULL, the data node will be detached from all hypertables. |
+| `force`       | Force detach of the data node even if that means that the replication factor is reduced below what was set. Note that it will never be allowed to reduce the replication factor below 1 since that would cause data loss.         |
+| `repartition` | Change the partitioning configuration so that all the remaining attached data nodes are used evenly. Defaults to true. |
```

Comment:
Same here as above:

`If set to true, makes the number of space partitions equal to the new number of data nodes (if such configuration exists). This ensures that the remaining data nodes are used evenly.`

Reply:
Went with `Makes the number of space partitions equal to the new number of data nodes (if such partitioning exists). This ensures that the remaining data nodes are used evenly. Defaults to true.`